### DISCUSSION TITLE
Scheduled Restart with kick & server "lock"

### DISCUSSION THREAD

**Discussion by LimitlessDrag0n** [gameserverapp/Platform#729] [Suggestions]
If possible: 
It would be great when a server has a scheduled shutdown, the server would kick any player like 2 Minutes before and locks the server for reentry to prevent duplicating (happening in DayZ) but I think, it would be great to have for other servers aswell set up.

In my case, for an example, I have a restart of my DayZ servers every 4 hours and when you just wait for the second the server restarts, you're able to duplicate items. This would or can be prevented with a kick and a lock of the server. 

Maybe I did not see that's already possible in the scheduled tasks. Maybe, I need to set up something differently?

**Comment by birdman3131** [comment]
https://commands.gg/7dtd/kickall This might work. Depends on the amount of time it takes to rejoin.
Run it 2 mins before the update.

**Reply by LimitlessDrag0n** [reply]
Thanks, tried it but this wont work. Maybe it's for 7D2D. I found a #lock and #unlock command that works but the kick, of the bohemia interactive wiki, only works with ID's...

**Reply by birdman3131** [reply]
Gah. Sorry about that. My brain swapped the games. That was not my intention.

**Reply by birdman3131** [reply]
https://forums.dayz.com/topic/227927-wanted-tool-to-kick-all-players/?do=findComment&comment=2297108 Supposedly you can use the player number as well. Not sure how many slots/players  you have but this might work.

**Reply by LimitlessDrag0n** [reply]
Yeah saw this also. Farther down the post you see there is a comment about "editing in C+ for that" <- not a beast with that tbh :D

I could use the scheduler for that matter... Okay, think I need to go this approach then. But thought, when we does have a tool that provides a restart trigger/function, why not include this also to this trigger
